app/assets/javascripts/admin/addon/templates/admin-badges/show.hbs

```diff
@@ -90,14 +90,14 @@
         </Content>
         <Content @name="upload-image">
           <form.Field
-            @name="image"
+            @name="image_url"
             @showTitle={{false}}
             @title={{i18n "admin.badges.image"}}
             @onSet={{this.onSetImage}}
             @onUnset={{this.onUnsetImage}}
             as |field|
           >
-            <field.Image />
+            <field.Image @type="badge_image" />
```

Comment:
This was what was causing the 422 error when presigning, every UppyImageUploader needs a type